### ISSUE TITLE
cx_freeze keeps looking in python/libs, not in compiled libs

### ISSUE THREAD

**Issue by El-Dragons** [marcelotduarte/cx_Freeze#654]
here is my setup.py 
from cx_Freeze import setup, Executable
import sys
import os
PYTHON_INSTALL_DIR = os.path.dirname(os.path.dirname(os.__file__))
os.environ['TCL_LIBRARY'] = os.path.join(PYTHON_INSTALL_DIR, 'tcl', 'tcl8.6')
os.environ['TK_LIBRARY'] = os.path.join(PYTHON_INSTALL_DIR, 'tcl', 'tk8.6')

# from setuptools import setup, find_packages
base = None

executables = [Executable("Master.py", base=base)]

packages = ['datetime','sys','pandas','os','openpyxl','unicodedata','idna','ctypes','unittest','numpy','matplotlib','nptdms']
options = {
'build_exe': {
    'packages':packages,
    'includes': ['matplotlib.backends.backend_tkagg','datetime','sys','pandas','os','openpyxl','unicodedata','idna','ctypes','unittest','numpy','matplotlib','nptdms'],
    # "excludes": ["tkinter","jinja2","jupyter_client","jupyter_core",
    # "html","tornado""logging",
    # "olefile","ipython_genutils","concurrent","pycparser","scipy","win32com",
    # "pkg_resources","lxml","PIL","_pytest"],
    'include_files': [(os.path.join(PYTHON_INSTALL_DIR, 'DLLs', 'tcl86t.dll'),
                                         os.path.join('lib', 'tcl86t.dll')),
                                        (os.path.join(PYTHON_INSTALL_DIR, 'DLLs', 'tk86t.dll'),
                                         os.path.join('lib', 'tk86t.dll'))
                                        # add here further files which need to be included as described in 1.
                                       ],
    "optimize": 1,
},
}
setup(
    name = "XXXXX",
    options = options,
    version = "3.0",
    description = 'XXXXXXXXXXXXXXXXXXXI ',
    executables=executables
)

in Main.py I have added prints to see where it is stuck. since it does not see any specific error

print(' importing sys ')
import sys
print(' importing numpy ')
import numpy as np 
print(' importing pandas ')
import pandas as pd
print(' importing TDMSplotter ')
from TdmsFile import TDMSplotter
print(' importing matplotlib')
import matplotlib.pyplot as plt
import unicodedata
import os
from openpyxl import load_workbook
from DTCFunctions import *

While running exe it prints running numpy and stops working

**Comment by marcelotduarte** [comment]
Take a look at issue #643 
You're using cx-freeze 6.1?
See my comments. Your setup has some includes, packages, etc that can be removed like the other user.

**Comment by El-Dragons** [comment]
Hello Sir,
I am new to python. I am learning as I am writing. 
my cx_freeze version is   6.1. 
I have attached py files I have been using. 
[files.zip](https://github.com/anthony-tuininga/cx_Freeze/files/4619903/files.zip)
after compilation, before running the '.exe' I rename the Annaconda3 folder to Annaconda31 so it will not be able to take anything from there and should be able to run stand-alone. 
Removed packages but still the same issue.
